controls/raml2html_works.rb

```diff
@@ -0,0 +1,36 @@
+title 'Tests to confirm raml2html binary works as expected'
+
+base_dir = input("base_dir", value: "bin")
+plan_origin = ENV['HAB_ORIGIN']
+plan_name = input("plan_name", value: "raml2html")
+plan_ident = "#{plan_origin}/#{plan_name}"
+
+control 'core-plans-raml2html' do
+  impact 1.0
+  title 'Ensure raml2html binary is working as expected'
+  desc '
+  We first check that the raml2html binary we expect is present and then run version checks on both to verify that it is excecutable.
+  '
+
+  hab_pkg_path = command("hab pkg path #{plan_ident}")
+  describe hab_pkg_path do
+    its('exit_status') { should eq 0 }
+    its('stdout') { should_not be_empty }
```

Comment:
Hi @MindNumbing.  What about an extra stderr check here:  its('stderr') { should be_empty }

Reply:
👍